src/content.ts

```diff
@@ -563,6 +577,8 @@ function sponsorsLookup(id: string) {
 
                 startSponsorSchedule();
             }
+
+            updateAdFlag();
```

Comment:
Is this check required here? If so, it should be moved above the `startSponsorSchedule()` call, since it uses the variable.

Reply:
Its required because it detects whether it changed from an ad playing or video playing. Its moved above startSponsorSchedule now